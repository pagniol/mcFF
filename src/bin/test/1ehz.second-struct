      
GCGGAUUUAGCUCAGUUGGGAGAGCGCCAGACUGAAGAUCUGGAGGUCCUGUGUUCGAUCCACAGAAUUCGCACCA

(((((((..((((........))))(((((((.....)))))))....((((((...)..))))))))))))....

(((((((..((((.....(..))))((((((.......))))))....(((((..)....))))))))))))....
(((((((..((((.....(..))))((((((.......))))))....((((((.).)..))))))))))))....
(((((((..((((.....(..))))((((((.......))))))....((((((.).)..))))))))))))....
((((((((.(((()(...(..))))((((((.......))))))...)((((((.).)..))))))))))))....
((((((((.(((()(...(..))))((((((.......))))))...)((((((.).)..))))))))))))....
(((((((..((((.(...(..))))((((((.......))))))...)((((((.).)..))))))))))))....
(((((((..((((.........)))((((((.......))))))....(((((....)..)))))))))))).... #enlever les loops de 2 et moins


(((((((..((((.....(..))))((((((.......))))))....((((((.).)..))))))))))))....
(((((((..((((........))))((((((.......))))))....((((((...)..))))))))))))....


Paires nom trouvées par aucun
    32:38

Paires trouvés par les deux et qui ne dois pas être retirées
    [19, 56, '++c; G-C Ww/Ww pairing antiparallel cis XIX']
