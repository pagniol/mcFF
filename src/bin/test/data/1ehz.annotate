Residue conformations -------------------------------------------
A1 : G C3p_endo anti
A2 : C C3p_endo anti
A3 : G C3p_endo anti
A4 : G C3p_endo anti
A5 : A C3p_endo anti
A6 : U C3p_endo anti
A7 : U C2p_endo anti
A8 : U C3p_endo anti
A9 : A C2p_endo syn
A10 : 2MG
A11 : C C3p_endo anti
A12 : U C3p_endo anti
A13 : C C3p_endo anti
A14 : A C2p_exo anti
A15 : G C3p_endo anti
A16 : H2U
A17 : H2U
A18 : G C2p_endo anti
A19 : G C2p_endo syn
A20 : G C3p_endo anti
A21 : A C3p_endo anti
A22 : G C2p_exo anti
A23 : A C3p_endo anti
A24 : G C3p_endo anti
A25 : C C3p_endo anti
A26 : M2G
A27 : C C3p_endo anti
A28 : C C3p_endo anti
A29 : A C3p_endo anti
A30 : G C3p_endo anti
A31 : A C3p_endo anti
A32 : OMC
A33 : U C3p_endo anti
A34 : OMG
A35 : A C3p_endo anti
A36 : A C4p_exo anti
A37 : YYG
A38 : A C3p_endo anti
A39 : PSU
A40 : 5MC
A41 : U C3p_endo anti
A42 : G C3p_endo anti
A43 : G C3p_endo anti
A44 : A C3p_endo anti
A45 : G C3p_endo anti
A46 : 7MG
A47 : U C2p_endo anti
A48 : C C2p_endo anti
A49 : 5MC
A50 : U C3p_endo anti
A51 : G C3p_endo anti
A52 : U C3p_endo anti
A53 : G C3p_endo anti
A54 : 5MU
A55 : PSU
A56 : C C3p_endo anti
A57 : G C3p_endo anti
A58 : 1MA
A59 : U C3p_endo anti
A60 : C C2p_endo anti
A61 : C C3p_endo anti
A62 : A C3p_endo anti
A63 : C C3p_endo anti
A64 : A C3p_endo anti
A65 : G C3p_endo anti
A66 : A C3p_endo anti
A67 : A C3p_endo anti
A68 : U C3p_endo anti
A69 : U C3p_endo anti
A70 : C C3p_endo anti
A71 : G C3p_endo anti
A72 : C C3p_endo anti
A73 : A C3p_endo anti
A74 : C C3p_endo anti
A75 : C C3p_endo anti
A76 : A C2p_endo anti
A590 : MG
A530 : MN
A510 : MG
A520 : MN
A540 : MG
A560 : MG
A570 : MG
A580 : MG
A550 : MN
A101 : HOH
A102 : HOH
A103 : HOH
A104 : HOH
A105 : HOH
A106 : HOH
A107 : HOH
A108 : HOH
A109 : HOH
A110 : HOH
A111 : HOH
A112 : HOH
A113 : HOH
A114 : HOH
A115 : HOH
A116 : HOH
A117 : HOH
A118 : HOH
A119 : HOH
A120 : HOH
A121 : HOH
A123 : HOH
A124 : HOH
A125 : HOH
A126 : HOH
A127 : HOH
A128 : HOH
A129 : HOH
A130 : HOH
A131 : HOH
A132 : HOH
A133 : HOH
A134 : HOH
A135 : HOH
A136 : HOH
A137 : HOH
A138 : HOH
A139 : HOH
A140 : HOH
A141 : HOH
A142 : HOH
A143 : HOH
A144 : HOH
A145 : HOH
A146 : HOH
A147 : HOH
A148 : HOH
A149 : HOH
A150 : HOH
A152 : HOH
A153 : HOH
A155 : HOH
A156 : HOH
A157 : HOH
A158 : HOH
A159 : HOH
A160 : HOH
A161 : HOH
A162 : HOH
A163 : HOH
A164 : HOH
A166 : HOH
A167 : HOH
A168 : HOH
A170 : HOH
A171 : HOH
A172 : HOH
A173 : HOH
A174 : HOH
A175 : HOH
A178 : HOH
A181 : HOH
A183 : HOH
A184 : HOH
A185 : HOH
A186 : HOH
A189 : HOH
A191 : HOH
A195 : HOH
A196 : HOH
A197 : HOH
A200 : HOH
A204 : HOH
A205 : HOH
A206 : HOH
A208 : HOH
A210 : HOH
A214 : HOH
A219 : HOH
A222 : HOH
A223 : HOH
A226 : HOH
A228 : HOH
A230 : HOH
A231 : HOH
A233 : HOH
A591 : HOH
A592 : HOH
A593 : HOH
A596 : HOH
A598 : HOH
A602 : HOH
A603 : HOH
A608 : HOH
A610 : HOH
A611 : HOH
A612 : HOH
A613 : HOH
A616 : HOH
A618 : HOH
A626 : HOH
A627 : HOH
A633 : HOH
A635 : HOH
A644 : HOH
A648 : HOH
A657 : HOH
A662 : HOH
A670 : HOH
A671 : HOH
A675 : HOH
A688 : HOH
A690 : HOH
A693 : HOH
A707 : HOH
A708 : HOH
A709 : HOH
A710 : HOH
A711 : HOH
A712 : HOH
A713 : HOH
A714 : HOH
A715 : HOH
A716 : HOH
A717 : HOH
A718 : HOH
A719 : HOH
A720 : HOH
A721 : HOH
A722 : HOH
A723 : HOH
A724 : HOH
A725 : HOH
A726 : HOH
A727 : HOH
A728 : HOH
A729 : HOH
A730 : HOH
A731 : HOH
A732 : HOH
A733 : HOH
A734 : HOH
A735 : HOH
A736 : HOH
A737 : HOH
A738 : HOH
A739 : HOH
A740 : HOH
A741 : HOH
A742 : HOH
Adjacent stackings ----------------------------------------------
A1-A2 : adjacent_5p upward 
A2-A3 : adjacent_5p upward 
A3-A4 : adjacent_5p upward 
A4-A5 : adjacent_5p upward 
A5-A6 : adjacent_5p upward 
A6-A7 : adjacent_5p upward 
A11-A12 : adjacent_5p upward 
A12-A13 : adjacent_5p upward 
A14-A15 : adjacent_5p upward 
A22-A23 : adjacent_5p upward 
A23-A24 : adjacent_5p upward 
A24-A25 : adjacent_5p upward 
A27-A28 : adjacent_5p upward 
A29-A30 : adjacent_5p upward 
A30-A31 : adjacent_5p upward 
A35-A36 : adjacent_5p upward 
A42-A43 : adjacent_5p upward 
A43-A44 : adjacent_5p upward 
A44-A45 : adjacent_5p upward 
A50-A51 : adjacent_5p upward 
A51-A52 : adjacent_5p upward 
A56-A57 : adjacent_5p upward 
A59-A60 : adjacent_5p upward 
A62-A63 : adjacent_5p upward 
A63-A64 : adjacent_5p upward 
A64-A65 : adjacent_5p upward 
A65-A66 : adjacent_5p upward 
A66-A67 : adjacent_5p upward 
A67-A68 : adjacent_5p upward 
A68-A69 : adjacent_5p upward 
A70-A71 : adjacent_5p upward 
A71-A72 : adjacent_5p upward 
A72-A73 : adjacent_5p upward 
A73-A74 : adjacent_5p upward 
A74-A75 : adjacent_5p upward 
Non-Adjacent stackings ------------------------------------------
A1-A73 : outward 
A3-A71 : outward 
A8-A13 : inward 
A9-A45 : inward 
A18-A57 : inward pairing 
A19-A57 : outward 
A21-A48 : outward 
A29-A42 : outward 
A51-A64 : outward 
A53-A62 : outward 
Number of stackings = 45
Number of adjacent stackings = 35
Number of non adjacent stackings = 10
Base-pairs ------------------------------------------------------
A1-A72 : G-C Ww/Ww pairing antiparallel cis XIX 
A2-A71 : C-G Ww/Ww pairing antiparallel cis XIX 
A3-A70 : G-C Ww/Ww pairing antiparallel cis XIX 
A4-A69 : G-U Ww/Ws pairing antiparallel cis XXVIII 
A5-A68 : A-U Ww/Ww pairing antiparallel cis XX 
A6-A67 : U-A Ww/Ww pairing antiparallel cis XX 
A7-A66 : U-A Ww/Ww pairing antiparallel cis XX 
A8-A14 : U-A Ww/Hh pairing antiparallel trans XXIV 
A8-A21 : U-A O2'/Ww pairing 
A9-A11 : A-C O2'/Bh pairing 
A9-A13 : A-C O2P/Bh pairing 
A9-A23 : A-A Hh/Hh Bh/O2P pairing parallel trans II 
A11-A24 : C-G Ww/Ww pairing antiparallel cis XIX 
A12-A23 : U-A Ww/Ww pairing antiparallel cis XX 
A13-A22 : C-G Ww/Ww pairing antiparallel cis XIX 
A15-A48 : G-C Ww/Ww pairing parallel trans XXII 
A18-A57 : G-G O2'/Ww O2'/Bs inward pairing 
A19-A56 : G-C Ww/Ww pairing antiparallel cis XIX 
A20-A21 : G-A Bs/O2' adjacent_5p pairing 
A27-A43 : C-G Ww/Ww pairing antiparallel cis XIX 
A28-A42 : C-G Ww/Ww pairing antiparallel cis XIX 
A29-A41 : A-U Ww/Ww pairing antiparallel cis XX 
A33-A35 : U-A O2'/Hh Hh/O2P Ww/O2P pairing 
A33-A36 : U-A Ww/O2P pairing 
A50-A64 : U-A Ww/Ww pairing antiparallel cis XX 
A51-A63 : G-C Ww/Ww pairing antiparallel cis XIX 
A52-A62 : U-A Ww/Ww pairing antiparallel cis XX 
A53-A61 : G-C Ww/Ww pairing antiparallel cis XIX 
A60-A61 : C-C O2P/Bh adjacent_5p pairing 