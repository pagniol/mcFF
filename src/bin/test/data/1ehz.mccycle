  245 nodes
106 edges
Minimum cycle basis size 31
[ A13R:C A8R:U A14R:A ] 3
[ A42R:G A29R:A A41R:U ] 3
[ A53R:G A61R:C A62R:A ] 3
[ A72R:C A1R:G A73R:A ] 3
[ A3R:G A2R:C A71R:G ] 3
[ A52R:U A53R:G A62R:A ] 3
[ A3R:G A70R:C A71R:G ] 3
[ A42R:G A28R:C A29R:A ] 3
[ A51R:G A63R:C A64R:A ] 3
[ A50R:U A51R:G A64R:A ] 3
[ A56R:C A19R:G A57R:G ] 3
[ A19R:G A18R:G A57R:G ] 3
[ A27R:C A28R:C A42R:G A43R:G ] 4
[ A51R:G A52R:U A62R:A A63R:C ] 4
[ A71R:G A72R:C A1R:G A2R:C ] 4
[ A69R:U A70R:C A3R:G A4R:G ] 4
[ A68R:U A69R:U A4R:G A5R:A ] 4
[ A67R:A A68R:U A5R:A A6R:U ] 4
[ A66R:A A67R:A A6R:U A7R:U ] 4
[ A23R:A A24R:G A11R:C A12R:U ] 4
[ A22R:G A23R:A A12R:U A13R:C ] 4
[ A11R:C A12R:U A23R:A A9R:A A102MG ] 5
[ A8R:U A13R:C A22R:G A23R:A A9R:A ] 5
[ A15R:G A48R:C A21R:A A22R:G A13R:C A14R:A ] 6
[ A8R:U A9R:A A45R:G A467MG A47R:U A48R:C A15R:G A14R:A ] 8
[ A17H2U A18R:G A19R:G A20R:G A21R:A A48R:C A15R:G A16H2U ] 8
[ A23R:A A9R:A A45R:G A44R:A A43R:G A27R:C A26M2G A25R:C A24R:G ] 9
[ A53R:G A545MU A55PSU A56R:C A57R:G A581MA A59R:U A60R:C A61R:C ] 9
[ A8R:U A7R:U A66R:A A65R:G A64R:A A50R:U A495MC A48R:C A15R:G A14R:A ] 10
[ A21R:A A48R:C A495MC A50R:U A51R:G A52R:U A53R:G A545MU A55PSU A56R:C A19R:G A20R:G ] 12
[ A31R:A A32OMC A33R:U A34OMG A35R:A A36R:A A37YYG A38R:A A39PSU A405MC A41R:U A29R:A A30R:G ] 13