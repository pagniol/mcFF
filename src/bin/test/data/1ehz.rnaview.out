Processing a single file ... 
 
PDB data file name: ../my-files/stage-E24/bin/test/1ehz.pdb_new 
 
########################################################RNA/DNA chain_ID: A from residue   1 to  76 
 
The backbone torsion angles are in ../my-files/stage-E24/bin/test/1ehz.pdb_new_torsion.out 
 
Finding all the base pairs... 
pair-type   1   1  72 ++c 
pair-type   2   2  71 ++c 
pair-type   3   3  70 ++c 
pair-type   4   4  69 WWc 
pair-type   5   5  68 --c 
pair-type   6   6  67 --c 
pair-type   7   7  66 --c 
pair-type   8   8  14 WHt 
pair-type   9   9  23 HHt 
pair-type  10  10  25 ++c 
pair-type  11  11  24 ++c 
pair-type  12  12  23 --c 
pair-type  13  13  22 ++c 
pair-type  14  15  48 WWt 
pair-type  15  16  59 SWt 
pair-type  16  18  55 WSt 
pair-type  17  19  56 ++c 
pair-type  18  22  46 HWt 
pair-type  19  26  44 WWc 
pair-type  20  27  43 ++c 
pair-type  21  28  42 ++c 
pair-type  22  29  41 --c 
pair-type  23  30  40 ++c 
pair-type  24  31  39 WWc 
pair-type  25  49  65 ++c 
pair-type  26  50  64 --c 
pair-type  27  51  63 ++c 
pair-type  28  52  62 --c 
pair-type  29  53  61 ++c 
pair-type  30  54  58 WHt 
 The total base pairs = 30 (from  76 bases); 
 Loop 1 is from residue 54 to residue 60. 
 Loop 2 is from residue 14 to residue 21. 
 Loop 3 is from residue 33 to residue 37. 
 Number of Helix =  2; The longest is number 1 
 
Ploting 2D structure... 
Input file is (../my-files/stage-E24/bin/test/1ehz.pdb.xml) with number of chains (1) 
helix_width =    59.65 
Number of base pairs (including tertiary interactions):  40 
Number of helix (anti-parallel):  4 
Number of single strand:  6 
 
2D structure (PS) finished! see output file: ../my-files/stage-E24/bin/test/1ehz.pdb.ps 
 
Time used: 0.02 seconds 
 
JOB finished! Time used: 0.02 seconds